Library/Formula/hfstospell.rb

```diff
@@ -0,0 +1,23 @@
+class Hfstospell < Formula
+  desc "Helsinki Finite-State Technology ospell"
+  homepage "http://www.ling.helsinki.fi/kieliteknologia/tutkimus/hfst/"
+  url "https://downloads.sourceforge.net/project/hfst/hfst/archive/hfstospell-0.3.0.tar.gz"
+  sha256 "07b5b368882cac2399edb1bb6e2dd91450b56f732c25413a19fcfe194342d70c"
+
+  depends_on "pkg-config" => :build
+  depends_on "libarchive"
+
+  def install
+    # Libxml++ headers do not seem to compile on El Capitan
```

Comment:
Does it work if you add `ENV.cxx11` at the beginning of the method and `needs :cxx11` above it? See e.g. [`libsass.rb`](https://github.com/Homebrew/homebrew/blob/a39f359529c826d64a54197895fb450f3d7cc40e/Library/Formula/libsass.rb#L17-L20) for an example.